system-test/spanner.ts

```diff
@@ -1411,7 +1421,11 @@ describe('Spanner', () => {
           )
           .then(data => {
             const rows = data[0];
-            assert.deepStrictEqual(rows.shift().toJSON(), EXPECTED_ROW);
+            assert.deepStrictEqual(
+              // tslint:disable-next-line: no-any
+              (rows as any).shift().toJSON(),
```

Comment:
And here!

Reply:
same thing here row is single object and we are using `shift()` will give compilation error.

Reply:
used 
```ts
const rows = data[0] as {} as Row[];
```